CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Breaking changes
 
+* `jj op abandon` now discards previous versions of a change (or predecessors)
+  if they become unreachable from the operation history. The evolution history
+  is truncated accordingly.
```

Comment:
Perhaps mention that "once this feature is used, jj <0.30 will not be able to read the repo". Or is the problem with jj<0.30 going to be subtler thn that?